### PR TITLE
Migrate from ttmath to stint

### PR DESCRIPTION
**Pull request by mratsim**

### CODE REVIEW

File: src/utils_numeric.nim [hunk 1]
```diff
@@ -5,33 +5,36 @@
...
 # at your option. This file may not be copied, modified, or distributed except according to those terms.
 
-import ttmath, constants, strformat, strutils, sequtils, endians, macros, utils / padding, rlp
+import stint, constants, strformat, strutils, sequtils, endians, macros, utils / padding, rlp
 
 # some methods based on py-evm utils/numeric
 
 # TODO improve
 
 proc intToBigEndian*(value: UInt256): Bytes =
```
Comment (yglukhov) [inline src/utils_numeric.nim]: These procs seem to know too much about stint internal structure. Can we just move them to stint lib?

File: src/vm/value.nim [hunk 2]
```diff
@@ -21,7 +21,7 @@ type
 
 # TODO: The Int256 value is stored as array[32, byte], and we bitcast it
 # back and forth. This is a hacky workaround for the problem that clang
-# doesn't let you store ttmath types inside nim variant types (unions). Things
+# doesn't let you store stint types inside nim variant types (unions). Things
```
Comment (yglukhov) [inline src/vm/value.nim]: I suppose this is no longer the case, so we can just revert this hack completely.

File: src/utils_numeric.nim (hunk 1 above)
Comment (mratsim) [reply src/utils_numeric.nim]: Yes I will move them this week, for now I just focused on porting 1-to-1 and just exposing Stint internal but tested functionality (like `countLeadingZeroBits`) and avoid introducing new procs/tests/reviews in Stint.

File: src/vm/value.nim (hunk 2 above)
Comment (mratsim) [reply src/vm/value.nim]: Good catch, it seems like only vint is used externally here: https://github.com/status-im/nimbus/blob/master/src/runner.nim#L86

Will remove that hack as well.

**Pull request merged**